spec/glueby/contract/timestamp/tx_builder/simple_spec.rb

```diff
@@ -63,4 +63,82 @@
       end
     end
   end
+
+  describe '#build' do
+    let(:wallet) { TestWallet.new(internal_wallet) }
+    let(:internal_wallet) { TestInternalWallet.new }
+    let(:fee_estimator) { Glueby::Contract::FeeEstimator::Fixed.new }
+    let(:prefix) { 'prefix' }
+    let(:content) { 'content' }
+    let(:builder) do
+      Glueby::Contract::Timestamp::TxBuilder::Simple
+        .new(wallet, fee_estimator)
+        .set_inputs(nil)
+        .set_data(prefix, content)
+    end
+    let(:unspents) do
+      2.times.map do |i|
+        {
+          txid: '1d49c8038943d37c2723c9c7a1c4ea5c3738a9bad5827ddc41e144ba6aef36db',
+          script_pubkey: '76a914234113b860822e68f9715d1957af28b8f5117ee288ac',
+          vout: i,
+          amount: 50_000_000,
+          finalized: true
+        }
+      end
+    end
+
+    before do
+      allow(Glueby::Wallet).to receive(:load).and_return(wallet)
+      allow(internal_wallet).to receive(:list_unspent).and_return(unspents)
+    end
+
+    context 'it gets a prefix that has a single hexadecimal digit' do
+      let(:prefix) { 'prefix' }
```

Comment:
制御文字を直接入れるとわかりにくいと思います。
```suggestion
      let(:prefix) { "prefix\x01" }
```